.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+exclude: ^(.idea|.vscode)/
```

Comment:
In the default usage, it will only check files being committed. You can run it manually which will check all files, but that will respect all the settings / config for the individual checks (like the pyyhon tools). I think all checks should ignore stuff in gitignore for example. We can explicitly ignore node_modules if you feel it is needed.
See https://pre-commit.com/ for the info.

Reply:
And of course you don't have to use it, you need to enable it separately for the checks to run

Reply:
Ah okay, thanks for clarifying 👍. Was wondering as to why there was that .vscode/.idea exclude but nothing else.